classes/class.ilMumieTaskHookPlugin.php

```diff
@@ -37,10 +37,9 @@ public function handleEvent($component, $event, $parameters)
             case 'beforeLogout':
                 $userId = $parameters["user_id"];
                 include_once('Customizing/global/plugins/Services/Repository/RepositoryObject/MumieTask/classes/class.ilMumieTaskSSOToken.php');
-                include_once('Customizing/global/plugins/Services/Repository/RepositoryObject/MumieTask/classes/class.ilMumieTaskIdHashingService.php');
-                $hashedId = ilMumieTaskIdHashingService::getHashForUser($userId);
-                if (ilMumieTaskSSOToken::tokenExistsForUser($hashedId)) {
-                    ilMumieTaskSSOToken::invalidateTokenForUser($hashedId);
+                if(ilMumieTaskSSOToken::invalidateAllTokenForUser($userId))
+                {
+                    ilLoggerFactory::getLogger('xmum')->info("LOGOUT FROM SERVER");
                     $this->logoutFromAllServers();
                 }
```

Comment:
the method name `invalidateAllTokenForUser ` doesn't indicate that it would have any kind of response type. Therefore, using it's result in an if condition can be confusing.

Also we shouldn't log unnecessarily.

I think we can actually get rid of the entire if statement and simply call `logoutFromAllServers` after calling `invalidateTokenForUser`